Lexplorer/Pages/Index.razor

```diff
@@ -1,7 +1,12 @@
 ﻿@using Lexplorer.Components;
 @page "/"
 
-
 <PageTitle>The Lexplorer - Home</PageTitle>
-<MudText Typo="Typo.caption">A Layer 2 block explorer for Loopring. This a work in progress, expect many outages as the Lexplorer is updated.</MudText> 
-<HomepageOverview></HomepageOverview>
+<MudContainer Fixed="true">
+    <MudItem lg="12" Class="mt-3">
+        <MudText Typo="Typo.h6">A Layer 2 block explorer for Loopring. This a work in progress, expect many outages as the Lexplorer is updated.</MudText> 
+    </MudItem>
+    <HomepageOverview />
+</MudContainer>
```

Comment:
I freely admit I have a very limited understanding of MudBlazor and it's breakpoints, so bear with me please.

1. Why does the single MudText need to be within a MudItem, i.e. what layout problem does this fix?
2. Why does this item only need the lg breakpoint - and not ones for xs and md as well?
3. The extra "mt-3" adds a margin at the top, correct?
4. With the extra container with fixed = true, you're achieving that the HomePageOverview (and the text above) are snapping to the different breakpoints, right? What's the advantage here? I would have thought the HomePageOverview component would handle the breakpoints itself just fine - and that they don't really matter for the text?

Reply:
MudBlazor breakpoint and in fact frontend as a whole is not my strong side so I am also learning as I go and it may not be that all my changes is the correct one even though it works as expected. There may be better way to reach the same result.
1. In the examples I have seen before and also what I have experienced myself is that putting different components into items makes it easier to set breakpoints as not all components have this option. 
2. When setting lg only it affects all/most screen sizes. If no different breakpoint is needed, its better to set one specific, be it lg, md, sm makes no difference. It just prevents it from being totally different if upgrading MudBlazor in my experience. 
3. mt-3 sets margin top, that's right. In some cases it does not work as expected, that's why it's not allways enough to set it on containerlevel but also on the specific component. MudBlazor is always developing. 
4. For ease of use it's best to set the component (index, overviews etc) within a container. And set it to Fixed will "snap" it to the closest breakpoint according to the documentation.

Reply:
1. OK I get that you use MudItem for breakpoints, when the individual component may not support them. But again - this just a simple text, right? Why does it even need to bother about breakpoints? As I originally asked: what layout problem did you see - *with that specific MudText* - that made you add an item? Sorry, if I'm coming through as irritating, but I honestly wanna know and learn.
2. Understood - I think we used "sm" before when we specified one for all breakpoints, so this is just a matter of preference then.
3. Okay, but then in later commits, I saw you using "my-3" and also "my3" - I presume that is for having margins at the top and bottom. Is there any difference for the ones with hyphen and without?
4. Again, I'm puzzled that it's up to the container hosting the component to care about breakpoints. I thought that's the job of the individual pages/components. For example, if we have a page like Transactions-Overview, it now always uses the full with, but within a container like your's it would snap to the different breakpoint widths. At least that's what I would expect, I'll need to try this myself. It would be great to see what problem this fixes in real-life, so if you could share a specific example, I'd really appreciate it.

Reply:
1. Haha no, I love how you are asking all the right questions. IIn that specific case I saw it break when going to mobile size display. That's why I tried to add it and it fixed the issue locally in my browser. 
2. Yeah it's a prefernce thing. 
3. If a have used my3, that is a typo. I will search and fix it. 
4. Yeah each component will then handle it's own breakpoints. To set it as Fixed will fix the issue when going to mobile displays as it sometimes even though the component has a breakpoint of sm=12 will not use the space available. It may be a bug with my browser (brave), but this fixed sizing also. If a component on small devices is set to 12, it should use that space with margins on both sides and not use half the space or full width without margins.

Reply:
As said before, this is not my project, so I don't want to step on any toes with my additions. It's all up to you :)

Reply:
> * If a have used my3, that is a typo. I will search and fix it.

It's here 4abf26c